content/docs/command-reference/exp/show.md

```diff
@@ -48,10 +46,10 @@ which you can typically exit by typing `Q`. Use `--no-pager` to print the table
 to standard output.
 
 By default, the printed experiments table will include columns for all metrics
-and params from the entire project. The `--param-deps`, `--include-metrics`, and
+and params from the entire project. The `--param-deps`, `--drop`, `--keep` and
```

Comment:
Unrelated to this PR, but why call out `--param-deps` first here? It seems like a less common option. Could `--only-changed` be called out first instead since it's often a useful option?